src/app/manager-dashboard/manager-dashboard.component.ts

```diff
@@ -1,12 +1,123 @@
-import { Component } from '@angular/core';
+import { Component, OnInit, ViewChild, AfterViewInit } from '@angular/core';
+import { CouchService } from '../shared/couchdb.service';
+import { Router } from '@angular/router';
+import { MatTableDataSource, MatSort, MatPaginator, MatDialog } from '@angular/material';
+import { DialogsDeleteComponent } from '../shared/dialogs/dialogs-delete.component';
+import { Location } from '@angular/common';
+import { Validators } from '@angular/forms';
+import { ValidatorService } from '../validators/validator.service';
+import { DialogsFormService } from '../shared/dialogs/dialogs-form.service';
+import { DialogsFormComponent } from '../shared/dialogs/dialogs-form.component';
+import { HttpClient } from '@angular/common/http';
+
 
 @Component({
-  template: `
-    <div><a routerLink="/community" i18n mat-raised-button>Communities</a> <a routerLink="/nation" i18n mat-raised-button>Nations</a></div>
-  `
+  templateUrl: './manager-dashboard.component.html'
 })
-export class ManagerDashboardComponent {
 
-  constructor() { }
+export class ManagerDashboardComponent implements OnInit, AfterViewInit {
+
+  constructor(
+    private couchService: CouchService,
+    private router: Router,
+    private dialog: MatDialog,
+    private location: Location,
+    private validatorService: ValidatorService,
+    private dialogsFormService: DialogsFormService,
+    private http: HttpClient
```

Comment:
`validatorService` and `http` are not used in this component and should be removed.